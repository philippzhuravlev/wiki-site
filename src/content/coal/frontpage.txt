# Coal Age

The Coal Age spans from 1800 to 1900 AUC, and follows from the [[steel/frontpage|Steel Era]]. The Coal Age is best characterized by the widespread use of coal as a fuel source, and the development of steam-powered machinery; magic has adapted to these changes, and is now used in industry and warfare. 
