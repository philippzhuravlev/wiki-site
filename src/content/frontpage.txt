# Welcome to the Wiki!

What if the gods were a bunch of etymology nerds and linguistic purists?

Urheimat (from German Urheimat, “original homeland”) is the working title name given to a fantasy/sci-fi world, heavily inspired by Dungeons and Dragons and Europa Universalis IV

This wiki is a work in progress and is not yet complete.

## How to use this wiki

This wiki is built with Next.js and React. It uses the [Markdown](https://www.markdownguide.org/) format for content.
