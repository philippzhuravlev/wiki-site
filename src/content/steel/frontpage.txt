# Steel Era

The Steel Era spans from 1000 to 1500 AUC. It was originally named due to the harsh and tough connotations of the word "steel", but it is now used to refer to the period of history in which steel production was widespread and used in industry and warfare; in reality, steel production was already known in the [[golden/frontpage|golden period]], but developments in metallurgy and the widespread use of steel in industry and warfare have made the name stick around.
