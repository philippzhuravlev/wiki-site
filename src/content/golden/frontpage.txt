# Golden Period

The Golden Period spans from 500 to 800 AUC. Settled civilization are by now ancient and well established, and so is the institutionalized use of magic in matters of governance, religion, warfare and industry.
